src/RFC-0313_VNRegistration.md

```diff
@@ -0,0 +1,138 @@
+The signature MUST as a schnorr type signature as follows:
+$$
+\begin{aligned}
+sig = r + e*k \\\\
+e = hash(R || K || Commitment) \\\\
+\end{aligned}
+$$
+As the challenge, we hash the nonce, VN public key, and the commitment. We include the commitment to ensure each signature must be unique. This is important
+as we don't want users to reuse all signatures. 
+
+
+## VN-Key
+
+The VN-key is the unique key used to identify where in the [shard space] this key is generated unique each time the VN is registered to assign the
+VN to a different area of the [shard space] each time.
+
+The key MUST be calculated as follows:
+
+$$
+\begin{aligned}
+\text{VN-key} = hash(\text{VN-pubkey}||\text{blockhash})
+\end{aligned}
+$$
+Where the `blockhash` is the blockhash of the block, the registration transaction was mined in.
+It is essential we ensure that the `VN-Key` has some random entropy inside of it, as the `VN-Key` needs to be pseudorandom.
+
+## VN-Key option two
+
+We only need to ensure that the `VN-key` has some random entropy inside of it. At first glance, the `blockhash` is an excellent easy source of random entropy that's different
+for each block. But this causes some issues with the proof in the block. If the `blockhash` is dependent on all the information in the block, we cannot include the current VN Merkle proof in the current block. 
+
+An easy fix for this would be to use another source of random entropy that does not depend on the entire current block info. We can replace the `blockhash` with the
+`utxo Merkle root`. This is also pseudorandom but does not rely on the `VN key` calculation. 
+$$
+\begin{aligned}
+\text{VN-key} = hash(\text{VN-pubkey}||\text{utxo merkle root})
+\end{aligned}
+$$
+
+
+## UTXO de-registration
+An option exists to lock the registration UTXO for a certain amount of blocks before allowing spending to ensure registration stays open. But it can be argued that this
+serves no real purpose as we don't care if the UTXO exists or not. We care that we have an accurate list of active VNs. Therefore we can say that we allow VNs to 
+de0register themselves by spending their registration UTXOs to a normal UTXO. We will still have the [delayed view] of the main chain, so this de-registration action will
+only take effect after some time. 
+
+## Epochs
+Radix's Cerberus uses epochs to manage VN stability and shuffling of VNs inside the [shard space]. It can be argued that we don't need to group nodes inside of 
+epochs to shuffle them around or lock them into a period of "service as a VN". Above in [UTXO de-registration], it was argued why we should not lock a VN to a 
+specific period in which they have to serve as VN. And in [Registration], it was argued why it is essential to force VNs to publish new registration transactions 
+periodically. We have also mentioned [delayed view] of the [Base Layer]. If we combine these three properties, we can argue that we have stability of the VN population.
```

Comment:
Rather be specific. Choose a term for this (maybe VNConfirmationPeriod), add it to the glossary and then update the RFCs to use it